### PR TITLE
Update jellyfish to 2.2.10

### PR DESCRIPTION
**Pull request by corburn**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by jerowe** [comment]
```
/opt/conda/conda-bld/jellyfish_1528918392008/work/Jellyfish-2.2.10 /opt/conda/conda-bld/jellyfish_1528918392008/work/Jellyfish-2.2.10
+ autoreconf -i
libtoolize: putting auxiliary files in '.'.
libtoolize: copying file './ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
libtoolize: copying file 'm4/libtool.m4'
libtoolize: copying file 'm4/ltoptions.m4'
libtoolize: copying file 'm4/ltsugar.m4'
libtoolize: copying file 'm4/ltversion.m4'
libtoolize: copying file 'm4/lt~obsolete.m4'
configure.ac:9: error: possibly undefined macro: AC_LIB_RPATH
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
configure.ac:32: error: possibly undefined macro: AC_LIB_LINKFLAGS_FROM_LIBS
autoreconf: /opt/conda/conda-bld/jellyfish_1528918392008/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/bin/autoconf failed with exit status: 1
```

File: recipes/jellyfish/meta.yaml [hunk 1]
```diff
@@ -1,30 +1,40 @@
...
-  url: https://github.com/gmarcais/Jellyfish/archive/v2.2.6.tar.gz
-  md5: 8a7ce28051171a4dc6b8f68cd39298c9
+  fn: v{{version}}.tar.gz
+  url: https://github.com/gmarcais/Jellyfish/archive/v{{version}}.tar.gz
+  sha256: {{sha256}}
+
 requirements:
   build:
-    - gcc
+    #- gcc #[linux]
```
Comment (jerowe) [inline recipes/jellyfish/meta.yaml]: Let's try with the linux-anvil gcc.

File: recipes/jellyfish/build.sh [hunk 2]
```diff
@@ -1,4 +1,6 @@
-#!/bin/bash
+#!/usr/bin/env bash
+
+set -x -e
```
Comment (jerowe) [inline recipes/jellyfish/build.sh]: FYI - using set -x -e will help with debugging with things go wrong. Its like setting a verbose flag in bash.

Comment (corburn) [reply recipes/jellyfish/build.sh]: Thanks for the tip 👍

**Comment by jerowe** [comment]
New errors!

```
config.status: executing libtool commands
+ make
  YAGGO    sub_commands/count_main_cmdline.hpp
ruby: error while loading shared libraries: libjemalloc.so.1: cannot open shared object file: No such file or directory
make: *** [sub_commands/count_main_cmdline.hpp] Error 127
```

**Comment by bgruening** [comment]
Awesome, thanks a lot!

**Pull request merged**